src/features/Lock/lockPeriod.js

```diff
@@ -135,7 +135,7 @@ const LockPeriod = ({
               }}
               onChange={handleOnSliderChange}
               value={lockValue}
-              max={remaining || 208}
+              max={remaining || 209}
```

Comment:
why 209?

Reply:
On the testnet it allowed me to use 209 weeks. But nvm, I just left it as it was